folsom/src/main/java/com/spotify/folsom/reconnect/ReconnectingClient.java

```diff
@@ -48,18 +52,20 @@
 
   private final BackoffFunction backoffFunction;
   private final ScheduledExecutorService scheduledExecutorService;
-  private final Connector connector;
+  private final com.spotify.folsom.reconnect.Connector connector;
```

Comment:
Is this needed?

Reply:
Yes, it's used in the retry method

Reply:
Sorry, I meant to fully qualified name

Reply:
Yes, there are two Connector classes (I guess we could rename one of them instead)